test/fixture/terraform_15_1.0/git_issue_88__corrupt_hcl_file/expected/main.terratag.tf

```diff
@@ -0,0 +1,31 @@
+provider "aws" {
+  version = "~> 2.0"
+  region  = "us-east-1"
+}
+
+resource "aws_s3_bucket" "b" {
+  bucket = "my-tf-test-bucket"
+  acl    = "private"
+
+  tags = merge(merge(tomap({
+    "Name" = "My bucket"
+  }), local.terratag_added_main), local.terratag_added_main)
```

Comment:
Notice this line. It's a marge of a merge. This is still a valid HCL file. Regardless, I will open a new ticket in case it's something to be fixed along the road.